src/main/java/io/vertx/redis/client/impl/RedisClusterConnection.java

```diff
@@ -138,23 +140,12 @@ public Future<Response> send(Request request) {
       case 0:
         // can run anywhere
         if (REDUCERS.containsKey(cmd)) {
-          final List<Future<Response>> responses = new ArrayList<>(slots.size());
-
-          for (int i = 0; i < slots.size(); i++) {
-            String[] endpoints = slots.endpointsForSlot(i);
-
-            final Promise<Response> p = vertx.promise();
-            send(selectMasterOrReplicaEndpoint(cmd.isReadOnly(args), endpoints, forceMasterEndpoint), RETRIES, req, p);
-            responses.add(p.future());
-          }
-
-          Future.all(responses).onComplete(composite -> {
-            if (composite.failed()) {
-              // means if one of the operations failed, then we can fail the handler
-              promise.fail(composite.cause());
-            } else {
-              promise.complete(REDUCERS.get(cmd).apply(composite.result().list()));
-            }
+          sendToAllSlots(promise, req, cmd, args, forceMasterEndpoint, REDUCERS.get(cmd));
+        } else if(cmd.equals(SCRIPT) && "LOAD".equalsIgnoreCase(new String(args.get(0)))) {
```

Comment:
```suggestion
        } else if (cmd.equals(SCRIPT) && args.size() > 1 && args.get(0).length == 4 && "LOAD".equalsIgnoreCase(new String(args.get(0)))) {
```

Nitpicking really, just for extra safety and to avoid needless allocation.